cfg/vars.cfg

```diff
@@ -59,10 +59,23 @@ export gitscripts_temp_path="${gitscripts_path}temp/"
 
 
 # formatting
-export H1HL="####################################################################################"
-export H2HL="------------------------------------------------------------------------------------"
-export H3HL="____________________________________________________________________________________"
-export EHL="!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+# export H1HL="####################################################################################"
+# export H2HL="------------------------------------------------------------------------------------"
+# export H3HL="____________________________________________________________________________________"
+# export EHL="!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+# export DELIMITER="${X}|${X}"
+
+
+## Print a horizontal rule
+rule () {
```

Comment:
Documentation here is pretty thin. Come on now, you need to re-harness those my-man docblocks!  😆 

But for real, at least adding the param(s) here would be nice. Even nicer would be an example with input/output...

Reply:
Pull requests welcomed.